packages/trader/src/App/Components/Elements/Modals/ServicesErrorModal/__tests__/services-error-modal.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ServicesErrorModal from '../services-error-modal';
+
+jest.mock('../authorization-required-modal.jsx', () => jest.fn(() => 'AuthorizationRequiredModal'));
+jest.mock('../insufficient-balance-modal.jsx', () => jest.fn(() => 'InsufficientBalanceModal'));
+jest.mock('../company-wide-limit-exceeded-modal.jsx', () => jest.fn(() => 'CompanyWideLimitExceededModal'));
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    return {
+        ...original_module,
+        Modal: jest.fn(() => <div data-testid='dt_mocked_modal' />),
+    };
+});
+
+describe('ServicesErrorModel component test cases', () => {
```

Comment:
I suggest to stick to general tests writing style. You may have a look in other packages (like cashier): `describe` include just the name of the component and each test case starts with lowercase. Example:
```
describe('<CashierSearchBox />', () => {
    const props = {
        onClear: jest.fn(),
        onSearch: jest.fn(),
        placeholder: 'Search placeholder',
        setIsSearchLoading: jest.fn(),
    };

    it('should have close icon if there is a text in input field', () => {
```

Reply:
Gotcha, will follow convention.